src/test/java/nextstep/subway/acceptance/StationAcceptanceTest.java

```diff
@@ -69,6 +69,25 @@ void createStation() {
     @DisplayName("지하철역을 조회한다.")
     @Test
     void getStations() {
+        // Given
+        Map<String, String> params = new HashMap<>();
+        params.put("name", "역삼역");
+        params.put("name2", "정자역");
+
+        RestAssured.given().log().all()
+                .body(params)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().post("/stations")
+                .then().log().all();
+
+        // When
+        List<String> stationNames = RestAssured.given().log().all()
+                .when().get("/stations")
+                .then().log().all()
+                .extract().jsonPath().getList("name", String.class);
+
+        // Then
+        assertThat(stationNames).containsAnyOf("역삼역", "정자역");
```

Comment:
`containsAnyOf` 를 사용하면 아래와 같이 하나의 요소만 포함되어 있어도 테스트를 통과합니다 😢 
`containsExactly` 나 `containsExactlyInAnyOrder` 같은 메서드를 사용하시는 걸 추천드립니다.

```suggestion
        assertThat(stationNames).containsAnyOf("역삼역", "홍대역");
```